apis/k8ssandra/v1alpha1/k8ssandracluster_webhook_test.go

```diff
@@ -233,6 +233,54 @@ func testNumTokens(t *testing.T) {
 	cluster.Spec.Cassandra.CassandraConfig.CassandraYaml.NumTokens = nil
 	err = k8sClient.Update(ctx, cluster)
 	require.Error(err)
+
+	// Num_token update validations
```

Comment:
Looks like the scenario involved in the bug still isn't covered. That is, an update to the K8ssandraCluster other than num_tokens should not fail on num_tokens.

Reply:
Good catch, I've updated w/ rebase to include that coverage in the latest push.